test/e2e/workload_clusters_test.go

```diff
@@ -36,6 +36,26 @@ func runWorkloadClusterFlowWithGitOps(test *framework.MulticlusterE2ETest, clust
 	test.DeleteManagementCluster()
 }
 
+func runWorkloadClusterFlowCheckRollingUpgrade(test *framework.MulticlusterE2ETest, clusterOpts ...framework.ClusterE2ETestOpt) {
+	latest := latestMinorRelease(test.T)
+	test.CreateManagementCluster(latest.Version, framework.ExecuteWithEksaRelease(latest))
+	test.RunInWorkloadClusters(func(w *framework.WorkloadCluster) {
+		w.GenerateClusterConfigForVersion(latest.Version)
+		w.CreateCluster(framework.ExecuteWithEksaRelease(latest))
+	})
+	time.Sleep(2 * time.Minute) // Time for workload MachineDeployments to stabilize
+	test.ManagementCluster.UpgradeCluster(clusterOpts)
+	time.Sleep(1 * time.Minute) // Time for new eks-a controller to kick in and potentially trigger rolling upgrade
```

Comment:
this is ok for now. with controller v2 we add attributes in eks-a objects status field to monitor upgrade status. we should later monitor them instead of sleep